tests/float32/basic.ml

```diff
@@ -0,0 +1,5 @@
+open Stdlib
+
+type f32 = float32
+
+type f32array = float32array
```

Comment:
Do we expect float32 to work correctly in the `ocaml/` compiler?  I think yes, in which case I think it's fine to add tests only to `ocaml/testsuite`.

Reply:
Hmm, I thought we'd decided that the unboxed types support wouldn't be available in `ocaml/`?

Reply:
float32 won't work in the `ocaml/` compiler. I'm not sure if there's a good way to disable the primitive types only in the upstream build, though.

Reply:
It seems fine not to support it.  I wish we had a way to test that it errors, but I don't see an easy way to.